_includes/extras.html

```diff
@@ -9,6 +9,10 @@
           {% icon github %} Edit on GitHub
         </a>
 
+        <a class="dropdown-item" href="{% link stats.md %}" title="Show view statistics about this rrepository">
+            {% icon galaxy-barchart %} Repository statistics
```

Comment:
```suggestion
            {% icon galaxy-barchart %} GTN statistics
```

not sure if 'repository' is confusing for end-users?